examples/advanced-routing/cafe-routes.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+metadata:
+  name: cafe
+spec:
+  hostnames:
+  - "cafe.example.com"
+  rules:
+  - backendRefs:
+    - name: main
+      port: 80
+---
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+metadata:
+  name: coffee
+spec:
+  hostnames:
+  - "cafe.example.com"
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /coffee
+      method: POST
+      headers:
+        - name: X-Demo-Header # header names and values are case-insensitive
```

Comment:
neat: inconsistent indentation (whitespace before `-`).

For example:
```yaml
spec:
  hostnames:
  - "cafe.example.com"
  rules:
  - matches:
```

And:
```yaml
      queryParams:
        - name: Great # query params and values are case-sensitive
          value: Example
```

Reply:
Fixed. I also updated my linter settings for yaml files to avoid this problem in the future.